src/test/__mocks__/providers/ServicesProvider.mock.tsx

```diff
@@ -28,6 +28,22 @@ export const lookupCacheService = {
   getById: jest.fn(),
 } as unknown as ILookupCacheService;
 
+export const recordNormalizingService = {
+  init: jest.fn(),
+  get: jest.fn(),
+} as IRecordNormalizingService;
```

Comment:
Perhaps we should adjust interface names to fully represent their purpose? Or at least make them consistent. Like here, some of them indicate that they're related to a service, some don't, and some contain only a partial reference to their functionality. What do you think?